### DISCUSSION TITLE
Allow accessing message payload for realtime in RLS policies

### DISCUSSION THREAD

**Discussion by ferhatwi** [supabase/supabase#28218] [Feature Requests]
Listening presence of users in a room can be done by this:
```js
const roomOne = supabase.channel('room_01')

const userStatus = {
  user: USER_ID,
  online_at: new Date().toISOString(),
}

roomOne.subscribe(async (status) => {
  if (status !== 'SUBSCRIBED') { return }

  const presenceTrackStatus = await roomOne.track(userStatus)
  console.log(presenceTrackStatus)
})
```
But this way, we can't be sure that `USER_ID` is indeed the id of the user that joins the presence channel, since we can't check if the value of the `user` key of payload equals `auth.uid()` in Row Level Security's insert policies.

**Comment by jacobrobertsbaca** [comment]
Bumping this
